developer-docs-site/docs/nodes/validator-node/index.md

```diff
@@ -61,12 +61,14 @@ To participate in the consensus process in the Aptos mainnet, you must deploy an
-- Bootstrap the nodes. The nodes now have the Aptos node binary running on them with the identity set. This fulfills the requirement for the Aptos network to become aware of your nodes. However, your nodes cannot connect to the Aptos network yet because these nodes are not yet in the validator set. On the Aptos network a validator can only accept another validator for connections. Until your nodes are in the validator set, they will be rejected by other validator nodes on the network. 
-- Perform the required actions before joining the validator set. For this, you must perform a few tasks such as initializing a staking pool, delegating to operators and voters, downloading the latest versions of the genesis blob and waypoint text files and restarting your nodes. 
-- Join the validator set. Other nodes will see your nodes and will establish connection to your nodes. Now you can stay in sync with the Aptos blockchain by building up your database of the history of the ledger. It takes some time for your nodes to build the database. Whenever your nodes reach the latest version of the blockchain, your validator node will be able to start participating in the consensus process.
+Follow the documents within this section to run a validator node and a validator fullnode in Aptos. Here is a summary of the process:
+
+1. Start by reading the node requirements to get to know the compute, memory and storage resources you need. Note also the internet bandwidth requirements. 
+1. Select a method to deploy your nodes, i.e., use a cloud service or Docker or source code. 
+1. Generate identity for the nodes. This is the first step in progressively making your nodes secure and ready to be integrated into the Aptos network. 
+1. Using YAML files, configure your nodes with user and network identity. This step enables the nodes to be recognized by other nodes in the Aptos network. Handshaking is possible after this step.  
+1. With the node identity established for the Aptos network, next you install the necessary binaries and locally generate the genesis blob and waypoint files. These will allow the node to be connected to the Aptos network. 
+1. Bootstrap the nodes. The nodes now have the Aptos node binary running on them with the identity set. This fulfills the requirement for the Aptos network to become aware of your nodes. However, your nodes cannot connect to the Aptos network yet because these nodes are not yet in the validator set. On the Aptos network a validator can only accept another validator for connections. Until your nodes are in the validator set, they will be rejected by other validator nodes on the network. 
+1. Perform the required actions before joining the validator set. For this, you must perform a few tasks such as initializing a staking pool, delegating to operators and voters, downloading the latest versions of the genesis blob and waypoint text files and restarting your nodes. 
```

Comment:
Probably group this step with the one above? The step above is purely informational and doesn't have any action associated with it, such as "ensure staking pool is initialized"

Reply:
Good point. I replaced this vague step with connecting to Aptos network. Let's further improve this in the next PR where we will link into related docs. Thanks, @rustielin !